docker/ubuntu/baseimage-18.04/Makefile

```diff
@@ -3,12 +3,12 @@ VERSION=0.3.0
 all: build
 
 build: Dockerfile bootstrap.sh
-	docker build --rm=true -t $(USER)/kobase-18.04:$(VERSION) .
+	docker build --rm=true -t $(REGISTRY)$(USER)/kobase-18.04:$(VERSION) .
```

Comment:
@NiLuJe I figure including `/` in `REGISTRY=this.that/` is nicer than some convoluted automated addition… unless I add it to the top of the Makefile perhaps. Thoughts?

Reply:
I just realized I can set the default REGISTRY to docker.io (or possibly something else) and always add a /. Much smarter.

Reply:
Plus the same thing in the Dockerfiles:
```diff
diff --git a/docker/ubuntu/baseimage-clang/Dockerfile b/docker/ubuntu/baseimage-clang/Dockerfile
index 84ef346..ee7e26e 100644
--- a/docker/ubuntu/baseimage-clang/Dockerfile
+++ b/docker/ubuntu/baseimage-clang/Dockerfile
@@ -1,4 +1,5 @@
-FROM koreader/kobase:0.3.0
+ARG REGISTRY=docker.io
+FROM $REGISTRY/koreader/kobase:0.3.0
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
diff --git a/docker/ubuntu/baseimage-clang/Makefile b/docker/ubuntu/baseimage-clang/Makefile
index 4230d82..cd8a617 100644
--- a/docker/ubuntu/baseimage-clang/Makefile
+++ b/docker/ubuntu/baseimage-clang/Makefile
@@ -1,14 +1,15 @@
 VERSION=0.3.0
+REGISTRY?=docker.io
 
 all: build
 
 build: Dockerfile
-       docker build --rm=true -t $(REGISTRY)$(USER)/kobase-clang:$(VERSION) .
+       docker build --build-arg REGISTRY=$(REGISTRY) --rm=true -t $(REGISTRY)/$(USER)/kobase-clang:$(VERSION) .
 
 shell:
-       docker run --detach-keys "ctrl-q,ctrl-q" --rm -t -i $(REGISTRY)$(USER)/kobase-clang:$(VERSION) bash -l
+       docker run --detach-keys "ctrl-q,ctrl-q" --rm -t -i $(REGISTRY)/$(USER)/kobase-clang:$(VERSION) bash -l
 
 push:
-       docker push $(REGISTRY)$(USER)/kobase-clang:$(VERSION)
+       docker push $(REGISTRY)/$(USER)/kobase-clang:$(VERSION)
 
 .PHONY: all clean test push shell
```